controllers/mobile/frmAgendaController/frmAgendaController.js

```diff
@@ -876,14 +876,16 @@ define({
                     this.view["speakerDescription" + speakerIndex].text = speakerBio.speaker_bio;
                     this.view["imgSpeaker" + speakerIndex].src = speakerBio.speaker_profile_pic;
                     this.view["ratingTile" + speakerIndex].setSpeakerProfileInRating(speakerBio);
+                  	this.view["ratingTile" + speakerIndex].setDefaultSelectedIndex();
                 }
             }
         }
         for (speakerIndex; speakerIndex < 3; speakerIndex++) {
             this.view["flxSpeaker" + speakerIndex].isVisible = false;
             this.view["ratingTile" + speakerIndex].isVisible = false;
         }
-    },
+        this.view["ratingTile"].setDefaultSelectedIndex();
```

Comment:
Please check the alignment of the label "Presentations" in Agenda detail

Reply:
Done